docfx/en/components/toc.json

```diff
@@ -562,7 +562,7 @@
         "status": ""
       },
       {
-        "exclude": ["Angular", "React", "WebComponents", "Blazor"],
+        "exclude": ["Angular", "React", "Blazor"],
```

Comment:
Don't we also want this topic for React?

Reply:
Yes. I am currently working on it